### ISSUE TITLE
GoodbyeDpi effect COD Warzone game

### ISSUE THREAD

**Issue by Wonyoung99** [ValdikSS/GoodbyeDPI#172]
It seems you can't play COD Warzone if Goodbyedpi installed in your pc. How can i fix it this problem. I dont want uninstall goodbyedpi everytime when i play game.

**Comment by ValdikSS** [comment]
See #148.
This seems to be an issue with the game/anti-cheat, not with GoodbyeDPI. Please contact game tech support and describe the issue in details to them. There's nothing I can do about it.

**Comment by ValdikSS** [comment]
Just reported the issue at https://support.activision.com/contact-us, but I'm not sure if this is a proper contact form. It did not ask for my email address, so I don't expect that the message will be read by technical personal. The message was as follows:

```
I'm the author of GoodbyeDPI anti-censorship software which bypasses government Internet blocks.

It seems that Call of Duty series does not launch if GoodbyeDPI is running, or even if it's service is present but not running. I assume that game anti-cheat engine detects GoodbyeDPI as a cheat, which it is not.

Please see reports from users that Call of Duty BO4, Call of Duty MW 2019 and Call of Duty Warzone does not start and they have to remove (not just stop!) the service:
https://github.com/ValdikSS/GoodbyeDPI/issues/172
https://github.com/ValdikSS/GoodbyeDPI/issues/148

The reporter of the issue has recorded the video, please also check it:
https://www.youtube.com/watch?v=eOViixWjNIQ

Please fix your anti-cheat detection. GoodbyeDPI is not a cheat.
Contact me at iam@valdikss.org.ru in case of additional questions, or write in these github issues.
```

**Comment by razcalzz** [comment]
it seems the anti-cheat detects WinDivert.dll as a cheat program, i've tested by disabling services:
1. sc stop "GoodbyeDPI" **unable to enter game** note: WinDivert1.4 is running
2. sc stop "GoodbyeDPI" + sc stop "WinDivert1.4" **able to play the game**

the game's anti-cheat does not like WinDivert1.4 it seems

**Comment by bugproof** [comment]
@razcalzz, of course, WinDivert can be used for cheating purposes e.g. to create ESP/radar hacks by intercepting packets with player positions or to modify packets as they go out to e.g. perfect silent aimbot, teleport hack etc. if the game is vulnerable to these kinds of hacks and server accepts it. These kinds of hacks aren't common because packets are usually encrypted and you need to reverse-engineer decryption logic and decrypt them first.

**Comment by Owyn** [comment]
I tried playing CoD long ago, it reacted the same way to macro program I use for my mouse AutoHotKey,
but all you had to do - was to rename its file and CoD would no longer react...
Perhaps the same way would help here.

**Comment by ValdikSS** [comment]
The developer of Easy Anti-Cheat reported that WinDivert should be unblocked, at least for RUST and SCUM game.
https://github.com/basil00/Divert/issues/153#issuecomment-421980793

Could anybody test whether this applies to Call of Duty?

**Comment by VladWinner** [comment]
> Could anybody test whether this applies to Call of Duty?

Актуальные Call of Duty из Battle.net используют другой, некий внутренний античит (а Warzone и последняя часть франшизы — Vanguard — перешли на RICOCHET "kernel-level anti-cheat"). С последней версией (0.2.2) не наблюдаю никаких проблем при работе через .bat, а вот в виде службы игра действительно вылетает через пару секунд.

**Comment by iDarkEmpire** [comment]
проблема актуальна на GoodbyeDPI v0.2.2

**Comment by iDarkEmpire** [comment]
> @iDarkEmpire
> 
> > проблема актуальна на GoodbyeDPI v0.2.2
> 
> Скачайте файы отсюда: https://github.com/ValdikSS/GoodbyeDPI/actions/runs/2070504898 Проявляется ли на них проблема?

да актуальна, проявляется, античит детектит и закрывает игру через 10 сек примерно

**Comment by alsnana** [comment]
> The developer of Easy Anti-Cheat reported that WinDivert should be unblocked, at least for RUST and SCUM game. [basil00/Divert#153 (comment)](https://github.com/basil00/Divert/issues/153#issuecomment-421980793)
> 
> Could anybody test whether this applies to Call of Duty?

Sorry for necroing about this.

Since the release of Warzone 2.0, trying to boot it up immediately crashes the game with no errors or anything (it made troubleshooting a nightmare and a huge waste of time to discover this program was the one holding the game from releaseing). I'm almost positive it effects Hunt: Showdown as well.

Seems like ending the both the service and goodbyedpi lets me play so I just made a batchfile to end goodbyedpi for the time being.

Annoying, but I guess that's modern anti cheat. I doubt it's goodbye's fault at this point. Thanks for this.

**Comment by SkyK0T** [comment]
Ёмае не увидел, и чего нет никакого решения ещё даже через костыли?

**Comment by Latgery** [comment]
Хелло. У меня после установки goodbyedpi заработал ютуб, но отвалилась возможность играть по сети в Dark souls 3 и Elden Ring. Конкретно в Элдене у меня случайным образом отваливается сеть, даже когда играю один. В Dark souls 3 я просто не могу подключиться к сети в главном меню. Как можно отключить goodbyedpi? 
Когда я использую вот эти команды: 
sc stop WinDivert1.4
sc delete WinDivert1.4
sc stop WinDivert
sc delete WinDivert
в командной строке пишет что служба не установлена/в настоящее время не принимает команды/служба отмечена для удаления

**Comment by smugone3131** [comment]
hi first time using github, but call of duty as of right now seems to work with goodbye dpi

<img width="135" height="49" alt="Image" src="https://github.com/user-attachments/assets/7e34abb9-37ff-4717-8f0c-cbbb18b6ccd0" />

**Comment by ItsSauron** [comment]
any fixes???? still can't play cod while using goodbyedpi in turkey
